### DISCUSSION TITLE
Can't assign alt-shift instead of ctrl-space in fcitx5

### DISCUSSION THREAD

**Discussion by burbilog** [fcitx/fcitx5#589] [Q&A]
Alt-shift is my "language" key for 20+ years. Ingrained into my brain, this reflex is kinda difficult to change. But I can't assign it instead of standard ctrl-space. Is it possible to fix? Single shift works. Single alt works. Alt-something works. But alt-shift does not....

Kubuntu 22.04.01, fcitx 5.0.14-1.

**Comment by wengxt** [comment]
What did you do to change this? I just test set alt shift in the place of ctrl space and it works?

Is it you set this but can't get it work, or just can't set it at all?

**Reply by burbilog** [reply]
> What did you do to change this? I just test set alt shift in the place of ctrl space and it works?
> Is it you set this but can't get it work, or just can't set it at all?

I found the source of the problem. System keyboard layout intercepts hotkey **before** fcitx5. I removed alt-shift in kde settings  (system settings -> input devices -> keyboard -> layouts -> main shortcuts and then fcitx5 was able to use this combination. Without that fcitx5 was unable to detect alt-shift at all. If you press alt first -- it detects left alt alone. If you press shift first -- it detects left shift. 

Please, add this to the FAQ. May it it's obvious, but it took a lot of time to figure out.
